soc/arm/nordic_nrf/nrf53/Kconfig.soc

```diff
@@ -89,6 +89,21 @@ config SOC_NRF5340_CPUNET_QKAA
 
 endchoice
 
+config SOC_DCDC_NRF53X_APP
```

Comment:
Can you add dependencies in these options, i.e. depend on SOC_NRF5340_CPUAPP

Reply:
done